alejandro-rodriguez-3115276934/tarea-6/main.js

```diff
@@ -0,0 +1,80 @@
+let numeros = [1, 2, 3, 4, 5];
+let numeros2=[];
+for(let i=0;i < 5;i ++){
+let resul=numeros[i]*2;
+  numeros2.push(resul);
+}
+ console.log(numeros2);
+
+ /* primer punto tarea 6 se recorre el array con un for y en casa iteracion se 
+ multiplica el valor por 2 */
+
+// =============================================================================
+ let productos = [
+    { nombre: "Camisa", precio: 20 },
+    { nombre: "Pantalón", precio: 30 },
+    { nombre: "Zapatos", precio: 50 },
+  ];
+  
+    for(const mayor in productos){
```

Comment:
el for aqui no esta haciendo nada, es totalmente innecesario si te fijas la variable mayor no esta siendo usada nunca...dos opciones o lo haces sin map y Math.min ya que vimos que no era necesario, o quitas el for...creeria que es mejor intentar la primera es decir sin usar map o Math.min